main.go

```diff
@@ -230,6 +231,10 @@ func visit(url *url.URL) {
 			ServerName:         host,
 			InsecureSkipVerify: insecure,
 		}
+
```

Comment:
The way I understand it, you have to explicitly opt-in using `http2.ConfigureTransport`.  If `TLSClientConfig` is non-nil, the stdlib will bail out of http/2 nego.  [See net/http/transport.go](https://github.com/golang/go/blob/a73020f847b3cf8575250569ebefb02573d19224/src/net/http/transport.go#L201)

As far as I know, the default transport has `TLSClientConfig: nil`.